go/test/endtoend/vtorc/readtopologyinstance/main_test.go

```diff
@@ -117,6 +117,7 @@ func TestReadTopologyInstanceBufferable(t *testing.T) {
 	assert.False(t, primaryInstance.HasReplicationCredentials)
 	assert.Equal(t, primaryInstance.ReplicationIOThreadState, inst.ReplicationThreadStateNoThread)
 	assert.Equal(t, primaryInstance.ReplicationSQLThreadState, inst.ReplicationThreadStateNoThread)
+	assert.Equal(t, fmt.Sprintf("%v:%v", keyspace.Name, shard0.Name), primaryInstance.ClusterName)
```

Comment:
This augments the existing `ReadTopologyInstanceBufferable` end to end test to also verify that we set the cluster_name correctly.